tests.py

```diff
@@ -2,25 +2,35 @@
 from pathlib import Path
 from tempfile import TemporaryDirectory
 
-from gitignore_parser import parse_gitignore
+from gitignore_parser import parse_gitignore, parse_gitignore_str
 
 from unittest import TestCase, main
 
 
 class Test(TestCase):
     def test_simple(self):
-        matches = _parse_gitignore_string(
+        matches = parse_gitignore_str(
             '__pycache__/\n'
             '*.py[cod]',
-            fake_base_dir='/home/michael'
+            base_dir='/home/michael/'
```

Comment:
Is the trailing slash in `/home/michael/` required?

Reply:
Done